templates/node--islandora-rdm-dataset--search-result.html.twig

```diff
@@ -0,0 +1,18 @@
+
+{% set created_date = node.getCreatedTime|format_date('umami_dates') %}
+{{ attach_library('classy/node') }} {# apply classy theme to node}
+                                    
+{# Format Creator (PublicationYear): Title. Publisher. (resourceTypeGeneral). Identifier #}
+<h4>{{ node.field_rdm_creator.0.entity.field_rdm_personal_name.value }} 
```

Comment:
The <h2 element should contain a class attribute like 'rdm-dataset-search-result-title' so that we can apply styling to it.